featurebyte/models/offline_store_feature_table.py

```diff
@@ -68,6 +79,7 @@ class OfflineStoreFeatureTableModel(FeatureByteCatalogBaseDocumentModel):
     output_dtypes: List[DBVarType]
     internal_entity_universe: Optional[Dict[str, Any]] = Field(alias="entity_universe")
     entity_lookup_info: Optional[EntityRelationshipInfo]
+    precomputed_lookup_feature_table_info: Optional[PrecomputedLookupFeatureTableInfo]
```

Comment:
To check my understanding, when this attribute it not empty, it means this table is derived from other existing offline feature table, right?

Reply:
Yes, that is correct.